DetectorDescription/DDCMS/src/DDFilteredView.cc

```diff
@@ -478,12 +479,12 @@ std::string_view DDFilteredView::get<string_view>(const string& key) const {
   int level = it_.back().GetLevel();
   for (auto const& i : refs) {
     auto k = find_if(begin(i->paths), end(i->paths), [&](auto const& j) {
-      auto const& names = split(j, "/");
-      int count = names.size();
+      auto frompos = j.find_last_of('/');
+      auto topos = j.size();
       bool flag = false;
-      for (int nit = level; count > 0 and nit > 0; --nit) {
+      for (int nit = level; frompos > topos and nit > 0; --nit) {
         std::string_view name = it_.back().GetNode(nit)->GetVolume()->GetName();
-        auto refname = names[--count];
+        std::string_view refname(j.substr(frompos + 1, topos));
```

Comment:
Just to make sure, is `j` a  `std::string_view`?

Reply:
yep, the compiler doesn't like `auto` here because it's being manipulated later